integreat_cms/api/v3/feedback/imprint_page_feedback.py

```diff
@@ -73,7 +73,7 @@ def imprint_page_feedback_internal(
     :return: JSON object according to APIv3 imprint feedback endpoint definition
     :rtype: ~django.http.JsonResponse
     """
-    if region.imprint:
+    if region.imprint and language in region.languages:
```

Comment:
```suggestion
    if region.imprint and language in region.visible_languages:
```
Probably it makes sense to only allow visible languages here.